webhook-connector/configure-the-webhook.md

```diff
@@ -4,50 +4,68 @@ description: Configurate the webhook in the Tradecloud portal
+### Orders Webhook Integration
 
-The shipment webhook configuration will be moved into the integration section in the future.
-{% endhint %}
+Enable the **Order Webhook Integration** if you want to receive a webhook trigger when an order has been issued or changed:
+
+![Order Events](../.gitbook/assets/webhook-order-events.png)
+
+A default set of order events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+
+{% page-ref page="order-events.md" %}
+
+![Order Delivery Schedule](../.gitbook/assets/webhook-order-delivery-schedule.png)
+
+In case of the order webhook you can choose between 
+* Multiple delivery lines per order line, also known as the **native** delivery schedule.
+* Only on delivery line per order line, also known as the **simple**  delivery schedule.
+
+See [the API manual](https://docs.tradecloud1.com/api/introduction/api/delivery-schedule) to read about the native versus the simple delivery schedule.
 
-### Select events to receive
+### Order Documents Webhook Integration
 
-A default set of events will be enabled. The actual set you want to receive wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+Enable the **Order Documents Webhook Integration** if you want to receive a webhook trigger when an order document has been attached:
 
-{% content-ref url="webhook-events.md" %}
-[webhook-events.md](webhook-events.md)
-{% endcontent-ref %}
+![Order Documents Events](../.gitbook/assets/webhook-order-documents-events.png)
 
-![](../.gitbook/assets/blue-sailboats-webhook-events.png)
+A default set of order document events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+
+{% page-ref page="order-documents-events.md" %}
+
+### Shipments Webhook Integration
+
+Enable the **Order Webhook Integration** if you want to receive a webhook trigger when a shipment has been issued or changed:
+
+![Shipment Events](../.gitbook/assets/webhook-shipment-events.png)
+
+A default set of shipments events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+
+{% page-ref page="shipment-events.md" %}
 
 ### Configure method, url and credentials
 
-![](../.gitbook/assets/blue-sailboats-webhook-url.png)
+![Order Webhook Congfig](../.gitbook/assets/webhook-order-config.png)
 
-* Select **GET**, **POST** or **PUT** as HTTP method.
-* Enter your webhook **URL**.&#x20;
+* Select **GET** or **POST** as HTTP method.
+* Enter your webhook **URL**
   * **https** is required.
-  * use **{orderId}** or **{legacyOrderId}** variables in the URL in case of **GET** method.
+  * use the **{orderId}** variable in the URL in case of **GET** method.
```

Comment:
```suggestion
  * You can use the **{orderId}** variable in the URL in case of **GET** method.
```

Reply:
Updated with `You must`

Reply:
I don't think it's required, right?

Reply:
It must be used, else you do not know which order has been triggered and the webhook makes no sense.